modules/shapes/assets/js/frontend/handlers/text-path.js

```diff
@@ -0,0 +1,217 @@
+export default class WordArtHandler extends elementorModules.frontend.handlers.Base {
```

Comment:
Please search-replace all formats.

```suggestion
export default class TextPathHandler extends elementorModules.frontend.handlers.Base {
```

Reply:
Yup, forgot to change that one..